vertical-pod-autoscaler/pkg/recommender/input/controller_fetcher/controller_fetcher.go

```diff
@@ -82,10 +83,41 @@ type controllerFetcher struct {
 	scaleNamespacer scale.ScalesGetter
 	mapper          apimeta.RESTMapper
 	informersMap    map[wellKnownController]cache.SharedIndexInformer
+	controllerCache controllerCacheStorage
+}
+
+func (f *controllerFetcher) periodicallyRemoveExpired(ctx context.Context, period time.Duration) {
```

Comment:
Here and below - since these functions are part of controllerFetcher and not controllerCache, could you please rename them so they include 'cache' or 'controllerCache' in their names?

Reply:
Done